### ISSUE TITLE
Load from Batch/PowerShell

### ISSUE THREAD

**Issue by dantman** [MehediH/winstall#47]
The Packs aren't great for managing a list of apps you use, to install on new systems. The format of .bat/.ps1 files with winget commands is pretty standard. Which a little bit of loose whitespace handling and ignoring extra args, it would probably be pretty easy to read winstall exported ones, ones exported by other sites, and even some manually authored ones.

It would be great to have a "Load" (or "Load from Batch/PowerShell") button that would let us choose say a `winstall.bat` file and have all the apps  in the file selected (with an option to override or merge if some are already selected).

Then it would be possible to download a `winstall.bat`, load it later, and download a new one - in order to essentially edit the `winstall.bat`.

**Comment by MehediH** [comment]
I am not sure exactly how Packs are bad for managing a list of apps to use on new systems, can you elaborate? 

I also don't understand how loading apps from the`winstall.bat` file in order to update your list of apps and creating a new batch file is any different than having a Pack, editing it whenever you need, and generating a new batch file?

**Comment by dantman** [comment]
> I am not sure exactly how Packs are bad for managing a list of apps to use on new systems, can you elaborate?
> having a Pack, editing it whenever you need, and generating a new batch file?

A few reasons, most of them around quirks in the current editing functionality. Right now the site has 2 sub-par editing experiences:

A) Use the Edit Pack feature: Adding new apps is limited to the fairly basic Add Apps dialog, you cannot use the /apps page which has pagination, uses the whole page, can be used to browse all apps and add things as you find them, and be sorted.
B) Use Select Apps then create a new app: This lets you use the whole /apps list; however now you have to re-do the name/description and delete all the old ones from each time you edit the list.

That edit pack experience combined with other small quirks (resource limits meaning you sometimes have to wait awhile before you can edit again, pointlessly public packs, and having to depend on one 3rd party service to stay online to keep your list of applications) make it sub par.

I know most of that can be fixed over time. However it's worth noting that the user probably already has a `winstall.bat`, a place they backup/store things they need to recreate their system they could store that file, and the "editing" experience would be as simple as "load winstall.bat, select/deselect apps, download winstall.bat, and overwrite old one".

**Comment by MehediH** [comment]
Yes, the Edit Packs experience is fairly limited at this point but ideally this wouldn't be the case, just trying to limit how much I have to spend on server costs for now. I think this can be easily improved, especially the public packs problem. But yes, adding apps to existing packs is something I have to rethink. 

It would be possible to make such a feature and I can see why that would be useful now, thank you! If you'd like to submit a PR for this, that wound be great :) Otherwise I will take a look when I get an opportunity.

**Comment by st-jliu** [comment]
This project was taken over by Splashtop now, we will continue the development and maintenance.

We have re-design the whole packs feature, and now you can persist the packs you made, locally or on the cloud.
You are welcome to provide further feedback and suggestions.
